zulipterminal/ui_tools/boxes.py

```diff
@@ -334,6 +338,11 @@ def main_view(self) -> List[Any]:
         else:
             recipient_header = None
 
+        # Dummy messages with just headers and no content
+        if self.empty_stream:
+            parts = [(recipient_header, recipient_header is not None)]
+            return [part for part, condition in parts if condition]
```

Comment:
Since i initially though we could create just a dummy message, which has only a recipient header, we don't need content, and other properties. Returning it here itself helps us avoid filling different properties of the dummy message like `author`, `timestamp`.
I did change it to `return [receipient header]` which does the same thing that code does.

Reply:
I'd need to read it again, and I'm off to bed, but I'm not sure that code is equivalent - if `recipient_header is not None` then we get `[recipient_header]` as you suggest, but when the condition is reversed I'm not sure they are equivalent.

Reply:
Yes. But i doubt the condition will ever get reversed if execution reaches till there. Anyway i'll check it again.